src/settings/StaffSlips/StaffSlipDetail.test.js

```diff
@@ -2,90 +2,80 @@ import React from 'react';
 import {
   render,
   screen,
-  within,
   fireEvent,
 } from '@testing-library/react';
 
 import '../../../test/jest/__mock__';
 
 import {
-  KeyValue,
-  Button,
+  Accordion,
+  AccordionSet,
+  ExpandAllButton,
 } from '@folio/stripes/components';
-import { PreviewModal } from '@folio/stripes-template-editor';
+import buildStripes from '../../../test/jest/__mock__/stripes.mock';
 
 import StaffSlipDetail from './StaffSlipDetail';
+import { Metadata } from '../components';
+import { StaffSlipAboutSection, StaffSlipTemplateContentSection } from './components/ViewSections';
```

Comment:
```suggestion
import {
  StaffSlipAboutSection,
  StaffSlipTemplateContentSection,
} from './components/ViewSections';
```

Reply:
Hi @artem-blazhko,
I think this is part of personal readability preferences.
It would be good to enforce such things through eslint rules.
However, I will be including it in my next commit.

Reply:
@Terala-Priyanka we try to use this approach through `ui-circulation` app. But you are definitely right it would be great to have eslint rule.
Thank you for fixing it.